bullmq/README.md

```diff
@@ -0,0 +1,50 @@
+# Judoscale for BullMQ
+
+Official [Judoscale](https://judoscale.com) adapter package for autoscaling BullMQ job queues.
+
+## Set up your app for autoscaling
+
+1. Install the judoscale-bullmq package:
```

Comment:
@carlosantoniodasilva Any thoughts on `judoscale-bullmq` vs `judoscale-bull-mq`?

This is our first Node adapter that's more than a single word. For some reason "bullmq" felt right to me when I was writing this, but it feels inconsistent with what we've done in other adapters.

Reply:
It seems that even though it's named "BullMQ", [their package is called `bullmq`](https://www.npmjs.com/package/bullmq):

![Screenshot 2024-05-22 at 16 10 28](https://github.com/judoscale/judoscale-node/assets/26328/1784d582-60c1-400b-b166-98eb116806f2)

It might feel inconsistent for someone using the judoscale version to have to import `bullmq` and follow that with `judoscale-bull-mq`, so I think it'd be okay to stick to `judoscale-bullmq` for keeping consistency with their package naming/convention.

Reply:
Great point! I feel much better. 😄